Z/zimg/build_tarballs.jl

```diff
@@ -0,0 +1,38 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "zimg"
+version = v"3.0.1"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/sekrit-twc/zimg/archive/refs/tags/release-3.0.1.tar.gz", "c50a0922f4adac4efad77427d13520ed89b8366eef0ef2fa379572951afcc73f")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/zimg-release-*
+./autogen.sh
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+make -j${nproc}
+make install
+install_license COPYING
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [p for p in supported_platforms() if p["arch"]!="aarch64"]
```

Comment:
```suggestion
platforms = filter!(p -> arch(p) != "aarch64", supported_platforms())
```
What's the problem with aarch64?